frontend/src/components/application/AppVendingControls/SetupControls.tsx

```diff
@@ -37,11 +37,10 @@ const SetupControls: FunctionComponent<Props> = ({ app, vendingConfig }) => {
   const [requirePayment, setRequirePayment] = useState(false)
 
   // Need existing app vending configuration to initialise controls
-  const {
-    status,
-    value: vendingSetup,
-    error,
-  } = useAsync(useCallback(() => getAppVendingSetup(app.id), [app.id]))
+  const vendingSetup = useQuery({
+    queryKey: ["appVendingSetup", app.id],
+    queryFn: () => vendingApi.getAppVendingSetupVendingappAppIdSetupGet(app.id),
```

Comment:
Credentials here is the last thing needed and then this LGTM

Reply:
Should be fixed now